src/lib/Libifl/int_manager.c

```diff
@@ -86,31 +103,58 @@ PBSD_manager(int c, int function, int command, int objtype, char *objname, struc
 			return pbs_errno;
 
 	/* now verify the attributes, if verification is enabled */
-	if ((pbs_verify_attributes(c, function, objtype,
-		command, aoplp)) != 0)
-		return pbs_errno;
-
-	/* lock pthread mutex here for this connection */
-	/* blocking call, waits for mutex release */
-	if (pbs_client_thread_lock_connection(c) != 0)
+	if ((pbs_verify_attributes(random_srv_conn(svr_connections), function, objtype, command, aoplp)))
```

Comment:
It would be better if we use pbs_verify_attributes_wrapper() here. It is because we can catch early if one of the servers is down i.e. this wrapper function returns PBSE_NOSERVER.